src/components/sections/SponsorSection.astro

```diff
@@ -0,0 +1,115 @@
+---
+
+import Sponsors from "../common/Sponsors/Sponsors.astro";
+import SubSectionHeading from "../common/Section/SubSectionHeading.astro";
+import SectionHeading from "../common/Section/SectionHeading.astro";
+import Section from "../common/Section/Section.astro";
+
+const sponsors = {
```

Comment:
While we are at refactoring this: can we put this into a JSON file? I need the identical information for the intro generator and so far have synchronised manually.

Reply:
done